src/ctapipe/utils/astro.py

```diff
@@ -18,8 +18,18 @@
 
 
 class StarCatalogues(Enum):
-    Yale = "V/50/catalog"  #: Yale bright star catalogue
-    Hipparcos = "I/239/hip_main"  #: hipparcos catalogue
+    Yale = {
```

Comment:
Why exactly is this needed?

Reply:
This loosely follows my suggestion to define applicable catalog metadata: the name/id of the catalog, the columns we are going to query/store (this is missing yet!), and some other metadata like frame. 
Then, the min amplitude cut can be applied on any column (and if attempted to apply to inexisting one, a KeyError should be thrown of course).

Reply:
Do we really need this, given that we query two catalogs? Again: I don't really see a use case for providing a general high-level wrapper around astroquery here.

I just wanted to have the tools to re-create these resource files so that developers can easily update them if needed.

Reply:
One has to define it somewhere. Right now it is done in the get_catalog functions, here looks more reasonable to me...